molgenis-app/src/main/java/org/molgenis/bbmri/controller/BbmriNlToEricMapperController.java

```
@@ -133,10 +138,17 @@ public void onApplicationEvent(ContextRefreshedEvent event)
 	{
 		try
 		{
-			doScheduleMappingJob();
+			doScheduleMappingJob(createCronTrigger(), "auto");
 		}
 		catch (Throwable t)
 		{
 		}
```

Do not silently catch, at least log the error